nautobot/extras/management/__init__.py

```diff
@@ -16,7 +16,6 @@
     "circuits.Circuit": circuit_choices.CircuitStatusChoices,
     "dcim.Cable": dcim_choices.CableStatusChoices,
     "dcim.Device": dcim_choices.DeviceStatusChoices,
-    "dcim.Site": dcim_choices.SiteStatusChoices,
```

Comment:
Just a note that this will intersect with the status migration changes in #3316. We'll need to coordinate depending on which gets merged first.